website/docs/how-to/how-to-create-feature-toggles.md

```diff
@@ -20,7 +20,7 @@ To be able to create a feature toggle in an Unleash system you will need:
 - A project to hold the toggle
 - A user with an **editor** or **admin** role OR a user with the following permissions inside the target project:
   - create feature toggles
-  - create/edit variants
+  - update variants (for the right environment)
```

Comment:
It's just the same as below, if it's unclear we should update the lines below

Reply:
Oh, interesting. Yeah, I see now that this is a how-to guide and not a reference document 😅  Yeah, in that case, I think this is fine..

Reply:
I also found it a bit misleading, because it says:
> To be able to create a feature toggle in an Unleash system you will need:
> - update variants 

and some other permissions which are not needed to create a feature toggle...